spring-cloud-gcp-samples/spring-cloud-spanner-r2dbc-samples/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <!-- Your own application should inherit from spring-boot-starter-parent -->
+    <artifactId>spring-cloud-gcp-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>spring-cloud-spanner-r2dbc-samples</artifactId>
+  <name>Spring Framework on Google Cloud Code Sample - Spanner-r2dbc</name>
+
+  <properties>
+    <sonar.skip>true</sonar.skip>
```

Comment:
Why?

Reply:
It came from the original repo, have removed it. Lets see if it changes anything in the sonar report